src/app/interfaces/StationStateUpdate.ts

```diff
@@ -0,0 +1,12 @@
+export interface StationStateUpdate {
+  type: "kStationStateUpdate";
+  value: {
+    callsign: string | undefined;
+    frequency: number;
+    rx: boolean;
+    tx: boolean;
+    xc: boolean;
+    xca: boolean;
+    headset: boolean;
```

Comment:
This now overlaps with the `FrequencyUpdate` type. They are quite similar and yet somewhat different. Also `xca` doesn't match with `crossCoupleAcross` used in `FrequencyUpdate`.

Seems like there's some sort of rationalization of the two types that needs to happen.